webpack/assets/javascripts/react_app/components/ConfigReports/Welcome.js

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+import { translate as __ } from '../../common/I18n';
+import EmptyState from '../common/EmptyState';
+import { getManualURL, getWikiURL } from '../../common/helpers';
+
+export const WelcomeConfigReports = () => {
+  const content = __(`If you wish to configure Puppet to forward it reports to Foreman, 
```

Comment:
Another grammar correction while we're here
```suggestion
  const content = __(`If you wish to configure Puppet to forward its reports to Foreman, 
```